lib/main.js

```diff
@@ -234,62 +314,86 @@ if (!process.env.CI) {
     })
   })
 
-  ava.test('graphics: boot/shutdown splash screen', async (test) => {
+  ava.test('graphics: Boot/shutdown splash screen', async (test) => {
+    console.log('\n\nGraphics: Shutdown splash screen')
     return utils.runManualTestCase(test, {
       prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
-      do: [ 'Reboot the device' ],
+      do: [ 'Shutdown the device' ],
```

Comment:
This test case covers graphics on BOTH shutdown and boot, so `Reboot` is actually correct.